homes_guests_loves.html

```diff
@@ -33,22 +33,25 @@ <h1 class="col-lg-12 col-sm-8">Homes guests loves</h1>
         <p class="home-hotel-name home-text">Hostel Friendship</p>
         <p class="home-destination home-text">Berlin, Germany <span class="hidden">destination</span></p>
                 </div>
-<!--        <div class="home-switcher">-->
-<!--            <img class="home-switcher-left" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">-->
-<!--            <img class="home-switcher-right" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">-->
-<!--        </div>-->
+        <div class="home-switcher">
+            <img class="home-switcher-left" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">
+            <img class="home-switcher-right" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">
+        </div>
 
     </div>
     </div>
 
 </section>
 
 <section class="best-destinations">
+    <div class="best-destinations-container">
     <h1 class="col-lg-12 col-sm-8">The best destinations</h1>
-    <form class="best-destinations-btns">
+    <form>
+            <div class="best-destinations-btns">
```

Comment:
it's better to set the last word "block" in this class name - if I see btn at the end of the class name I think that there should be button tag

Reply:
done